scripts/visualize/visualizeWalleyeDecline.R

```diff
@@ -1,28 +1,158 @@
 # // document.getElementById('data-line').getTotalLength(); then set that value to 'stroke-dasharray', 
 # // animate 'stroke-dashoffset' from that value down to 0. 
+library(XML)
 svgWallyDecline <- function(object, filename){
   line.i <- which(names(object$view.1.2) == 'lines')
+  ids <- c('walleye-line','bass-line')
   for (i in line.i){
     object$view.1.2[[i]]$class = 'data-line'
+    object$view.1.2[[i]]$id = ids[1]
+    ids <- tail(ids,-1)
   }
-  
-  object$css <- '#tick-labels, #y-title {
+  object$css <- '#tick-labels, #y-title, text {
   \tfont-family: Arial;
 }
 .data-line {
 \tstroke-linejoin: round;
 \tstroke-width:3;
+\tstroke-linecap: round;
 }
 #tick-labels {
-\tfont-size: 12.00pt; 
+\tfont-size: 10.00pt; 
 }
-#y-title {
+#y-title, #x-title {
 \tfont-size: 14.00pt; 
-}'
+}
+#walleye-line {
+  stroke-dasharray: 1125.464;
+  stroke-dashoffset: 1125.464;
+  animation: dash-walleye 12s linear forwards;
+}
+
+#bass-line {
+  stroke-dasharray: 1570.7915;
+  stroke-dashoffset: 1570.7915;
+  animation: dash-bass 12s linear forwards;
+}
+.background-walleye {
+       fill:#b2e7e2;
+       stroke:white;
+       stroke-width:2;
+       fill-opacity:0.75;
+    }
+.background-bass {
+       fill:#e0b2b2;
+       stroke:white;
+       stroke-width:2;
+       fill-opacity:0.75;
+}
+@keyframes shift-bass {
+	  0%   {transform: translateX(820px)}
+	  50% {transform: translateX(700px)}
+    100% {transform: translateX(720px)}
+}
+@keyframes background-largemouth-bass {
+	  0%   {transform: translate(0)}
+	  50% {transform: translate(-10px,5px)}
+    100% {transform: translateY(0)}
+}
+#all-bass {
+  animation: shift-bass 12s linear forwards;
+}
+#bass-1,#bass-19 {
+	  animation: background-largemouth-bass 2s ease infinite;
+	}
+	
+@keyframes dash-walleye {
+0%  {stroke-dashoffset: 1002.88890337959}
+4.76190476190476%  {stroke-dashoffset: 971.257314089763}
```

Comment:
@mwernimont I think switching over to keyframes makes this IE compliant (unlike the way we animated lines in the Lower CO drought viz)